src/web-ui/src/components/DemoGuideBadge/DemoGuideBadge.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <button @click="onClick" :aria-label="copy" class="demo-guide-badge d-inline-flex align-items-center text-left">
+    <div class="logo mr-1"><img :src="serviceLogo" alt="" class="img-fluid" /></div>
+    <div :class="{ text: true, 'hide-text-on-small-screens': hideTextOnSmallScreens }">
+      <div>{{ copy }}</div>
+      <div class="powered-by">
+        powered by <span class="service">{{ poweredByService }}</span>
+      </div>
+    </div>
+  </button>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+import { Articles } from '@/partials/AppModal/DemoGuide/config';
+
+const Services = {
+  Pinpoint: 'Pinpoint',
+  Personalize: 'Personalize',
+};
+
+export default {
+  name: 'DemoGuideBadge',
+  props: {
+    article: { type: String, required: true },
+    hideTextOnSmallScreens: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    service() {
+      switch (this.article) {
+        case Articles.TEXT_MESSAGING:
+        case Articles.PERSONALIZED_EMAILS:
+          return Services.Pinpoint;
+        case Articles.USER_PERSONALIZATION:
+        case Articles.PERSONALIZED_RANKING:
+        case Articles.SIMILAR_ITEM_RECOMMENDATIONS:
+          return Services.Personalize;
+      }
+
+      throw new Error('Invalid article passed to DemoGuideBadge');
+    },
+    serviceLogo() {
+      switch (this.service) {
+        case Services.Pinpoint:
+          return '/pinpoint.svg';
+        case Services.Personalize:
+          return '/personalize.svg';
+      }
+
+      throw new Error('Invalid article passed to DemoGuideBadge');
+    },
+    copy() {
+      switch (this.article) {
+        case Articles.TEXT_MESSAGING:
+          return 'Click here to learn more about Personalized product recommendations via SMS';
+        case Articles.USER_PERSONALIZATION:
+          return 'Click here to learn more about User Personalization';
+        case Articles.PERSONALIZED_RANKING:
+          return 'Click here to learn more about Personalized rankings';
+        case Articles.SIMILAR_ITEM_RECOMMENDATIONS:
+          return 'Click here to learn more about Similar item recommendations';
+        case Articles.PERSONALIZED_EMAILS:
```

Comment:
This might be something to discuss with James, but in Personalized emails, we have 2 different types available
1. Personalized recommendations email sent one time
2. Abandoned cart email which you have mentioned here.

I am wondering if we need to  speak about 1 here too. Deferring to @james-jory

Reply:
The personalized recommendations email is not triggered by any user interaction, not is it mentioned anywhere on the front end. Should it still be mentioned in the demo guide? Either way, I'm not sure it should be mentioned in the badge displayed next to the trigger abandon cart email button.